lib/miq_expression/field.rb

```diff
@@ -82,8 +82,20 @@ def contains(other)
     )
   end
 
+  def column_type
+    if custom_attribute_column?
+      CustomAttribute.find_by(:name => custom_attribute_column_name).field_type
```

Comment:
Is it possible that there are multiple instances of a custom attribute with the same name with values that have different classes? I'm not sure what can be done about it if that's the case, though.